GraknConsole.java

```diff
@@ -261,19 +254,31 @@ private void runQuery(GraknClient.Transaction tx, String queryString) {
             terminal.handle(Terminal.Signal.INT, s -> answerPrintingJob.cancel(true));
             answerPrintingJob.get();
             Instant end = Instant.now();
-            printer.info("answers: " + counter + ", duration: " + Duration.between(start, end).toMillis() +" ms");
-        } catch (InterruptedException | ExecutionException e) {
+            printer.info("answers: " + counter + ", duration: " + Duration.between(start, end).toMillis() + " ms");
+        } catch (InterruptedException e) {
             e.printStackTrace();
+        } catch (ExecutionException e) {
+            throw (RuntimeException)e.getCause();
         } catch (CancellationException e) {
             Instant end = Instant.now();
-            printer.info("answers: " + counter + ", duration: " + Duration.between(start, end).toMillis() +" ms");
-
+            printer.info("answers: " + counter + ", duration: " + Duration.between(start, end).toMillis() + " ms");
             printer.info("The query has been cancelled. It may take some time for the cancellation to finish on the server side.");
         } finally {
             terminal.handle(Terminal.Signal.INT, Terminal.SignalHandler.SIG_IGN);
         }
     }
 
+    private void shutdownQueryJobs() {
+        executorService.shutdown();
+        try {
+            if (!executorService.awaitTermination(100, TimeUnit.MILLISECONDS)) {
+                executorService.shutdownNow();
+            }
+        } catch (InterruptedException e2) {
```

Comment:
doesn't have to be e2

Reply:
right @lriuui0x0 ? right now, the implementation is like this

1. try to shut down nicely
2. if we can't, shut down forcefully

I think there's no point in attempting to shut down nicely since in the end we may just shut down forcefully anyway. Might as well just shut down forcefully from the beginning. It won't cause any thread or memory leaks.

```
private void shutdownQueryJobs() {
  executorService.shutdownNow();
}
```

Reply:
And then we can just inline `shutdownQueryJobs` since it only has one line

Reply:
@lolski I agree.